test/fixtures/correct.js

```diff
@@ -198,16 +198,16 @@ for (let semiSpacing = 0; semiSpacing < 10; ++semiSpacing) {
 }
 
 // `sort-imports`.
-import SortImport1 from 'sort-import1';
-import { sortImport2 } from 'sort-import2';
-import sortImport4 from 'sort-import4';
-import { sortImport3, sortImport5 } from 'sort-import';
-
-noop(SortImport1);
-noop(sortImport2);
+import { SortImport5, sortImport4 } from 'sort-import1';
+import { sortImport3 } from 'sort-import2';
+import SortImport2 from 'sort-import3';
+import sortImport1 from 'sort-import4';
```

Comment:
Just added. I think it's unnecessary to add those cases to the incorrect fixtures since it's not asserted which line incorrect, right?